plugins/spark/flytekitplugins/spark/task.py

```diff
@@ -76,11 +76,13 @@ def __init__(self, task_config: Spark, task_function: Callable, **kwargs):
         )
 
     def get_custom(self, settings: SerializationSettings) -> Dict[str, Any]:
-        from flytekit.bin import entrypoint as _entrypoint
+        spark_exec_path = settings.entrypoint_settings.path
+        if spark_exec_path is None:
```

Comment:
lets drop this? This is the new code, we should just assume this is always set?

Reply:
else actually better maybe we should default to this path? and then if we realize that we are not `in-container-serialize` then we should use `/opt/venv/bin/entrypoint.py`

Reply:
what do you mean?

Reply:
@kumare3 should I add a mode or some bool to SerializationSettings so we can set whether we are in container or not and whether to use the entrypoint path or the abs path?